<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RigGuiBaseQt</class>
 <widget class="QWidget" name="RigGuiBaseQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generic Rig Gui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="refereceHeader" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>67</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>67</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="headerLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="refLabel">
          <property name="text">
           <string>Active Rig Namespace:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="refPicker"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="newRefBut">
          <property name="text">
           <string>Add Namespace from Selection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delRefBut">
          <property name="text">
           <string>Remove Namespace</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="selectionLayout"/>
   </item>
   <item>
    <widget class="QWidget" name="toolsLayout" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="toolsRadioLayout" stretch="1,1,1,1">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <item>
         <widget class="QLabel" name="toolLabel">
          <property name="text">
           <string>Tools:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toolCheckBody">
          <property name="text">
           <string>Body Only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toolCheckFace">
          <property name="text">
           <string>Face Only</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toolCheckBoth">
          <property name="text">
           <string>Both</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="selectionToolsLayout" stretch="0,0,0,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="resetSelection">
          <property name="text">
           <string>Reset Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectAll">
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectUnkeyed">
          <property name="text">
           <string>Select Unkeyed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="invertSelection">
          <property name="text">
           <string>Invert Selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="reflectPoseGrp">
        <property name="title">
         <string>Reflect Pose for Selected Controls</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QPushButton" name="flipPose">
           <property name="text">
            <string>Flip Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mirrorPoseLtR">
           <property name="text">
            <string>Mirror L --&gt;&gt; R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mirrorPoseRtL">
           <property name="text">
            <string>Mirror R --&gt;&gt; L</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
